src/main/java/edu/greenblitz/tobyDetermined/subsystems/swerve/SwerveChassis.java

```diff
@@ -57,8 +57,8 @@ public static SwerveChassis getInstance() {
 	
 	@Override
 	public void periodic() {
-		updatePoseEstimation();
-		field.setRobotPose(getRobotPose());
+//		updatePoseEstimation();
```

Comment:
Commented code

Reply:
do we need to update the pose estimation twice?

Reply:
ok we dont